### ISSUE TITLE
check fails when it shouldn't (no vale errors, not supposed to fail on error)

### ISSUE THREAD

**Issue by andie787** [errata-ai/vale-action#106]
Even if all vale rules are set to suggestion, and fail on error is false, reviewdog logs everything as ERROR and the check fails with reporter github-check-pr.

Can you set the reviewdog error level in this action or do you need a different config file?

UPDATE: what I'm noticing is that my custom rules are running fine and output INFO level logs, the non-custom rule violations (from proselint and Google for example) output ERROR level and cause the check to fail

**Comment by jdkato** [comment]
> Even if all vale rules are set to suggestion,

Are you also setting the "non-custom" rules to suggestion?

**Comment by andie787** [comment]
Yes, everything was set to suggestion to try to ease into things

**Comment by jdkato** [comment]
`MinAlertLevel` is a display floor, not a severity rewrite — a rule that ships `level: error` still reports an error. Your custom rules are written `level: suggestion`; Google's and proselint's aren't.

Assign the level per rule:

    [*.md]
    BasedOnStyles = Google, proselint
    Google.Passive = suggestion

What the check concludes comes from the `level` input (`warning` → neutral). The action was ignoring it; fixed on `refresh`.
